### DISCUSSION TITLE
External data

### DISCUSSION THREAD

**Discussion by joegomes** [snap-stanford/ogb#369] [PCQM4M-LSC]
Hello OGB team! In the rules for the LSC, it is stated that "We do not allow the use of any external datasets to train models". Does this include the use of unlabeled data? Thanks for your time and attention.

**Comment by weihua916** [comment]
Hello Joe! For LSC datasets, we do not allow the use of any external data (including unlabeled data) because the data is already quite large and we want to avoid any potential hacking to win the competition (apply DFT on unlabeled data to increase the label supervision etc). For OGB datasets, we allow the use of unlabeled data. If fact, some entries are pre-trained on the LSC data (see [here](https://ogb.stanford.edu/docs/leader_graphprop/#ogbg-molpcba)).

**Reply by joegomes** [reply]
Hello Weihua! I appreciate your clarification of the rules. We will stick with the 2D/3D data provided for our work. Thanks for putting together this great competition together with the rest of the OGB team.
